sdk/identity/identity/src/credentials/managedIdentityCredential/index.ts

```diff
@@ -15,7 +15,7 @@ import { formatSuccess, formatError, credentialLogger } from "../../util/logging
 import { tracingClient } from "../../util/tracing.js";
 import { imdsMsi } from "./imdsMsi.js";
 import { tokenExchangeMsi } from "./tokenExchangeMsi.js";
-import { mapScopesToResource } from "./utils.js";
+import { mapScopesToResource, serviceFabricErrorString } from "./utils.js";
```

Comment:
total nit, feel free to ignore, but `serviceFabricErrorMessage` might be more descriptive